lib/badge-cli.spec.js

```diff
@@ -13,28 +13,30 @@ function runCli (args) {
 describe('The CLI', function () {
   it('should provide a help message', function () {
     return runCli([]).then(stdout => {
-      assert(stdout.startsWith('Usage'));
+      expect(stdout).to.startWith('Usage');
```

Comment:
I can run all tests in `lib` dir (`node_modules/.bin/mocha 'lib/*.spec.js'`) but running only this one test ends with an error:
```
> node_modules/.bin/mocha lib/badge-cli.spec.js
...
4 passing (2s)
  1 failing

  1) The CLI
       should provide a help message:
     Error: Invalid Chai property: startWith
      at Object.proxyGetter [as get] (node_modules/chai/lib/chai/utils/proxify.js:66:17)
      at runCli.then.stdout (lib/badge-cli.spec.js:16:24)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
```
I like to run tests from one spec (file) from IDE.

Reply:
Does adding `require('./register-chai-plugins.spec')` fix it?

Reply:
Yea, confirmed it does. Will push that change.

It's not easy to ensure these will always work that way, though it will always be an easy fix.